packages/@o3r/pipeline/schematics/ng-add/templates/github/__dot__github/workflows/main.yml

```diff
@@ -0,0 +1,54 @@
+name: Main CI
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/*'
+  pull_request:
+    branches:
+      - main
+      - 'release/*'
+
+jobs:
+  build:
+    runs-on: <%= runner %>
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: ./.github/actions/setup
+      - name: Build
+        run: <%= packageManager %> run build
+
+  code-checks:
+    runs-on: <%= runner %>
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: ./.github/actions/setup
+      - name: Unit tests
+        run: <%= packageManager %> run test
+      - name: Audit
+        run: <% if (packageManager === 'yarn') {%>yarn <%}%>npm audit
```

Comment:
Why not using our own action `AmadeusITGroup/otter/tools/github-actions/audit` ?

Reply:
Our action only supports yarn.

Reply:
In case of yarn we could use our action then and open an issue to support npm

Reply:
I agree that the audit action should support `npm` along with `yarn`. 
Now that I think about it, the audit step should not be here as it is supposed to be part of the security workflow.

Reply:
https://github.com/AmadeusITGroup/otter/issues/2292 created